core/domain/voiceover_services.py

```diff
@@ -30,47 +31,47 @@
     models.Names.VOICEOVER])
 
 
-def _get_entity_voiceover_from_model(
-    entity_voiceover_model: voiceover_models.EntityVoiceoversModel
-) -> voiceover_domain.EntityVoiceover:
-    """Returns the EntityVoiceover domain object from its model representation
+def _get_entity_voiceovers_from_model(
+    entity_voiceovers_model: voiceover_models.EntityVoiceoversModel
+) -> voiceover_domain.EntityVoiceovers:
+    """Returns the EntityVoiceovers domain object from its model representation
     (EntityVoiceoversModel).
 
     Args:
-        entity_voiceover_model: EntityVoiceoversModel. An instance of
+        entity_voiceovers_model: EntityVoiceoversModel. An instance of
             EntityVoiceoversModel.
 
     Returns:
-        EntityVoiceover. An instance of EntityVoiceover object, created from
+        EntityVoiceovers. An instance of EntityVoiceovers object, created from
         its model.
     """
     content_id_to_voiceovers_dict = {}
     for content_id, voiceover_type_to_voiceover in (
-            entity_voiceover_model.voiceovers.items()):
+            entity_voiceovers_model.voiceovers.items()):
         content_id_to_voiceovers_dict[content_id] = {
-            'manual': state_domain.Voiceover.from_dict(
-                voiceover_type_to_voiceover['manual']),
-            'auto': state_domain.Voiceover.from_dict(
-                voiceover_type_to_voiceover['auto'])
+            feconf.VoiceoverType.MANUAL: state_domain.Voiceover.from_dict(
```

Comment:
Same note about using a dict comprehension to generalize

Reply:
Addressed here
https://github.com/oppia/oppia/pull/19440/files#r1451761848